config/_default/menus/menus.en.yaml

```diff
@@ -0,0 +1,1263 @@
+main:
+
+#####################
+## GETTING STARTED ##
+#####################
+
+  - name: "Getting Started"
+    identifier: "getting_started"
+    url: "getting_started/"
+    pre: "nav_start"
+    weight: 10000
+
+  - name: "Agent"
+    identifier: "getting_started_agent"
+    url: "getting_started/agent/"
+    parent: "getting_started"
+    weight: 1
+
+  - name: "Datadog"
+    url: "getting_started/application/"
+    parent: "getting_started"
+    weight: 2
+
+  ## GETTING STARTED - Integrations
+
+  - name: "Integrations"
+    identifier: "getting_started_integrations"
+    url: "getting_started/integrations/"
+    parent: "getting_started"
+    weight: 3
+  - name: "Prometheus/Openmetrics"
+    identifier: "getting_started_prometheus"
+    url: "getting_started/prometheus/"
+    parent: "getting_started_integrations"
+    weight: 301
```

Comment:
Link doesn't work in the nav, check the preview.

Reply:
fixed